cli/proc_state.rs

```diff
@@ -495,7 +495,10 @@ impl ProcState {
     referrer: &str,
     permissions: &mut PermissionsContainer,
   ) -> Result<ModuleSpecifier, AnyError> {
-    if let Ok(referrer) = deno_core::resolve_url_or_path_deprecated(referrer) {
+    // TODO(bartlomieju): ideally we shouldn't need to call `current_dir()` on each
```

Comment:
**praise:** Not knowing the specifics here, but passing "base path" as argument sounds very much like sound logic.

Reply:
Yeah, that was the goal of the 6 PRs I opened since yesterday. `deno_core` no longer relies on calling `current_dir()` itself.